src/offline.js

```diff
@@ -6,7 +6,26 @@ export function init (cozy, { options = {}, docTypes = [] }) {
   }
 }
 
-export function addDocType (cozy, docType, options) {
+export function addDocType (cozy, docType, options = {}) {
   cozy._offline_databases = cozy._offline_databases || []
   cozy._offline_databases[docType] = new PouchDB(docType, options)
 }
+
+export function replicateFromCozy (cozy, docType, options = {}, events = {}) {
+  if (cozy._offline_databases && docType in cozy._offline_databases) {
+    const url = cozy._url + '/data/' + docType
+    let db = cozy._offline_databases[docType]
+    let replication = db.replicate.from(url, options)
+    const eventNames = [
+      'change', 'paused', 'active', 'denied', 'complete', 'error'
+    ]
+    for (let eventName of eventNames) {
```

Comment:
if you return the replication, I am not sure we need the events: the using code can call .on(x) itself.